apps/wasm-standalone/wasm-graph/tools/build_graph_lib.py

```diff
@@ -72,7 +72,7 @@ def build_graph_lib(opt_level):
     shape_dict = {input_name: img_data.shape}
 
     mod, params = relay.frontend.from_onnx(onnx_model, shape_dict)
-    target = "llvm -mtriple=wasm32-unknown-unknown -mattr=+simd128 --system-lib"
+    target = "llvm -mtriple=wasm32-unknown-unknown -mattr=+simd128"
```

Comment:
i think we need to keep --system-lib passed to Executor config here. can you audit the other places you removed `--system-lib` and ensure the Executor passed to relay.build has it?